config/controller.yaml

```diff
@@ -14,27 +14,27 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: tekton-pipelines-controller
+  name: controller
   namespace: tekton-pipelines
   labels:
     app.kubernetes.io/name: tekton-pipelines
     app.kubernetes.io/component: controller
-    tekton.dev/release: "devel"
+    pipeline.tekton.dev/release: "devel"
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: tekton-pipelines-controller
+      app: controller
```

Comment:
I didn't think changing this was upgrade-safe 🤔

Reply:
Good point :sweat_smile: I could (and probably should revert that change), but I though, if we wanna break people, let's break them now instead of real soon when we are beta & co…

/cc @sbwsg @bobcatfish @afrittoli

Reply:
Just noting that this change would break operator as well; see https://github.com/tektoncd/operator/blob/master/pkg/controller/setup/constants.go